docs/documentation/schedule/components/base.md

```diff
@@ -28,30 +30,19 @@ This feature allows to represent the basic information elements used to identify
 ## Routes
 
 <div class="grid" markdown>
-This feature helps define the elements that identify a transit route, including its name, description and the type of service that is being represented (such as a bus, a subway or metro, ferry, etc.).
+A route is a group of trips under the same branding that are displayed to riders as a single service. Each route has, among other attributes, ra name that matches the agency's rider-facing materials, and the type of service that is being represented (such as a bus, a subway or metro, ferry, etc.).
```

Comment:
Typo: "Each route has, among other attributes, **A** name that..."

Reply:
fixed in last commit